File: WebApplication.Net/Areas/Customers/Controllers/CustomerVMController.cs
using Microsoft.AspNetCore.Mvc;

using WebApplication.Net.Customers.Models;

namespace WebApplication.Net.Customers.Controllers
{
    [Area("Customers")]
    public class CustomerVMController : Controller
    {
        //
        // GET: /Customers/Customer/DisplayCustomer
        public ActionResult DisplayCustomer()
        {
            CustomerViewModel obj = new CustomerViewModel();
            obj.TxtName = "Panoramix";
            obj.TxtAmount = "1000";
            return View(obj);
        }
    }
}

File: WebApplicationFSharp.Net/Areas/Customers/Controllers/CustomerVMController.fs
namespace WebApplicationFSharp.Net.Customers.Controllers

open Microsoft.AspNetCore.Mvc

open WebApplicationFSharp.Net.Customers.Models

[<Area("Customers")>]
type CustomerVMController () =
    inherit Controller()

    let _customer = CustomerViewModel (TxtName = "Panoramix", TxtAmount = "1000")

    // GET: /Customers/CustomerVM/DisplayCustomer
    member this.DisplayCustomer () =
        _customer |> this.View :> ActionResult
